modules/stack/variables.tf

```
@@ -310,3 +310,33 @@ variable "description" {
   description = "Specify description of stack"
   default     = null
 }
+
+variable "dedicated_space" {
```

For booleans, we usually end them in `_enabled` unless they are pass-thru to a resource-level input of the same name.

```suggestion
variable "dedicated_space_enabled" {
```

---

cc: @aknysh thoughts?